### PR TITLE
remove dependencies on netclient

### PR DESCRIPTION
**Pull request by mattkasun**

### CODE REVIEW

**Comment by theguy951357** [comment]
netmaker-1 came back on this one 
Screenshot 2023-01-27 at 5 36 38 PM

**Review by abhishek9686 — COMMENTED** [review]
left comments on related PR https://github.com/gravitl/netclient/pull/164

**Comment by theguy951357** [comment]
will wait on changes

**Review by theguy951357 — APPROVED** [review]
everything functions as before

File: metrics/metrics.go [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+}
+
+// ResetMetricForNode - resets node level metrics
+func ResetMetricForNode(server, peerKey, peerID string) {
+	metric := GetMetric(server, peerKey)
+	delete(metric.NodeConnectionStatus, peerID)
+	UpdateMetric(server, peerKey, &metric)
+}
+
+const MetricCollectionInterval = time.Second * 25
```
Comment (0xdcarns) [inline metrics/metrics.go:85]: I'm sure it was missing before, but public variable missing comment

File: models/proxy.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	Peers           []wgtypes.PeerConfig   `json:"peers"`
+	PeerMap         map[string]PeerConf    `json:"peer_map"`
+	IsIngress       bool                   `json:"is_ingress"`
+	IsRelayed       bool                   `json:"is_relayed"`
+	RelayedTo       *net.UDPAddr           `json:"relayed_to"`
+	IsRelay         bool                   `json:"is_relay"`
+	RelayedPeerConf map[string]RelayedConf `json:"relayed_conf"`
+}
+
+// Metric - struct for metric data
```
Comment (0xdcarns) [inline models/proxy.go:74]: incorrect comment

**Review by 0xdcarns — CHANGES_REQUESTED** [review]

**Review by 0xdcarns — APPROVED** [review]

**Review by abhishek9686 — APPROVED** [review]

**Pull request merged**